src/js/view/controls/rightclick.js

```diff
@@ -62,6 +62,12 @@ export default class RightClick {
                 type: 'keyboardShortcuts'
             });
         }
+        if (!OS.mobile && model.get('pipIcon') !== 'disabled' && (Browser.chrome || Browser.edge || Browser.safari)) {
```

Comment:
Could set
`this.pipMenu = !OS.mobile && model.get('pipIcon') !== 'disabled' && (Browser.chrome || Browser.edge || Browser.safari)`
if we wanted, but LGTM!

Reply:
Thank you! I will make this change before merging